modules/installation-ibm-z-kvm-user-infra-machines-iso.adoc

```diff
@@ -0,0 +1,56 @@
+// Module included in the following assemblies:
+//
+// * installing/installing_ibm_z/installing-ibm-z-kvm.adoc
+
+[id="installation-user-infra-machines-iso-ibm-z_kvm_{context}"]
+= Creating {op-system-first} machines
+
+Before you install a cluster on IBM Z infrastructure that you provision, you must install {op-system} as {op-system-base-full} guest virtual machines for the cluster to use. Complete the following steps to create the machines.
+
+.Prerequisites
+
+* At least 1 LPAR running {op-system-base} 8.3 with KVM, referred to as {op-system-base} KVM host in this procedure.
+* A domain name server (DNS) that can perform hostname and reverse lookup for the nodes. 
+
+.Procedure
+
+. Install the KVM/QEMU hypervisor on the {op-system-base} KVM host.
+
+. Obtain the {op-system-base} QCOW2 disk image file from the link:https://access.redhat.com/downloads/content/290[Product Downloads] page on the Red Hat customer portal or the link:https://mirror.openshift.com/pub/openshift-v4/s390x/dependencies/rhcos/4.2/latest/[{op-system} image mirror] page.
++
+[IMPORTANT]
+====
+The {op-system} images might not change with every release of {product-title}.
+You must download images with the highest version that is less than or equal
+to the {product-title} version that you install. Only use the appropriate {op-system} QCOW2 image described in the following procedure.
+====
++
+. Download the QCOW2 disk image and Ignition files to a common directory on the {op-system-base} KVM host.
++
+For example: `/var/lib/libvirt/images`
++
+[NOTE]
+====
+The Ignition files are generated by the {product-title} installer. 
+====
+. Create a new disk image with the QCOW2 disk image backing file for each KVM guest node.
++
+----
```

Comment:
Should this take place in the terminal?

If so, maybe:
[source,terminal]
----
$ ....